platform/core/tests/test_m36_optimalisator.py

```
@@ -0,0 +1,1430 @@
+    """FRAVÆR NUMMER TO: døra finnes ikke.
+
+    En rettighet uten en dør og en dør uten en rettighet er hver for
+    seg en HALV sperre. Porten leser begge, og denne leser koden —
+    uten kommentarer og strenger, så den måler mønsteret og ikke
+    forklaringen på hvorfor mønsteret er unngått.
+    """
+    for fil in (MIGRASJON, *MODULFILER, FLATE):
+        kode = _bare_kode(fil, uten_strenger=True).lower()
+        for tabell in POLICYTABELLER:
+            for verb in ("insert into " + tabell, "update " + tabell,
+                         "delete from " + tabell):
```

Ruff (0.16.3)

[error] Possible SQL injection vector through string-based query construction

(S608)